src/plugins/paginate/paginate.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * @title WET-BOEW Tag filter
+ * @overview Filter based content tagging
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, document, wb ) {
+"use strict";
+
+let i18n, i18nText;
+
+const componentName = "wb-paginate",
+	selector = ".provisional." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	filterClass = "wb-fltr-out",
+	pgFilterOutClass = "wb-pgfltr-out",
+	tagFilterClass = "wb-tgfltr-out",
+	pagerClass = "wb-paginate-pager",
+	pageData = "data-pagination-idx",
+	notFilterClassSel = ":not(." + filterClass + "):not(." + tagFilterClass + ")",
+	defaults = {
+		lst: {
+			selector: "li"
+		},
+		grp: {
+			selector: "> *"
+		},
+		tbl: {
+			selector: "tr",
+			section: ":scope > tbody"
+		}
+	},
+
+	init = function( event ) {
+		const elm = wb.init( event, componentName, selector );
+
+		if ( elm ) {
+			var userConf = JSON.parse( elm.getAttribute( "data-wb-paginate" ) ),
+				paginationElm,
+				elmTagName = elm.nodeName,
+				setDefault,
+				uiTargetElm;
+
+			if ( !i18nText ) {
+				i18n = wb.i18n;
+				i18nText = {
+					prv: i18n( "prv" ),
+					nxt: i18n( "nxt" )
+				};
+			}
+
+			// Setup configurations
+			switch ( elmTagName ) {
+			case "UL":
+				setDefault = defaults.lst;
+				break;
+			case "TABLE":
+				setDefault = defaults.tbl;
+				break;
+			default:
+				setDefault = defaults.grp;
+				break;
+			}
+
+			elm.id = elm.id || wb.getId();
+			elm.pgSettings = { ...setDefault, ...userConf };
+			elm.pgSettings.currPage = 1;
+			elm.pgSettings.itemsPerPage = elm.pgSettings.itemsPerPage || 25;
```

Comment:
I suggest to move the default "25" at the top of the plugin js code. 

Comment: Would 7, 10 or 12 be a better choice compared to 25? That way it will avoid a content scrolling when a page is selected. The data table do have a default of 10 items. I know it is a minimal scrolling in wet-boew but it's going to be more prominent in GCWeb because the font are different.

Reply:
At this point, it is very subjective, but I'll set the default as 10 and add it to the defaults object.